src/main/scala/chisel3/util/Conditional.scala

```diff
@@ -24,10 +24,16 @@ object unless {  // scalastyle:ignore object.name
   * user-facing API.
   */
 class SwitchContext[T <: Bits](cond: T) {
+  private var whenContext: Option[WhenContext] = None
```

Comment:
is it possible to fold over all the ises without tracking mutable state (for an indeterminate time)?